### PR TITLE
(fleet/rook-ceph-conf) add rsphome export to pillan

### PR DESCRIPTION
**Pull request by cbarria**
forgot to merge the rsphome before so now it is been added to the fleet deployment

### CODE REVIEW

**Review by jhoblitt — APPROVED** [review]
LGTM

File: fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      failureDomain: host
+      replicated:
+        size: 3
+      quotas:
+        maxSize: 25Gi
+    - name: ec
+      failureDomain: host
+      erasureCoded:
+        dataChunks: 6
+        codingChunks: 3
```
Comment (jhoblitt) [inline fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml:25]: As pillan is a 9 node cluster, 6+3 is probably excess as a single node going down would prevent writes.   K + M <= nodes - 1

Comment (cbarria) [reply fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml:25]: but i think this cannot be changed now, right?

File: fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    - name: default
+      failureDomain: host
+      replicated:
+        size: 3
+      quotas:
+        maxSize: 25Gi
+    - name: ec
+      failureDomain: host
+      erasureCoded:
+        dataChunks: 6
```
Comment (jhoblitt) [inline fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml:24]:
```suggestion
        dataChunks: 4
```

File: fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml (hunk 1 above)
Comment (jhoblitt) [reply fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml:25]: No because 6+3 = 9

Comment (cbarria) [reply fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml:25]: this was changed 👍🏻

File: fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml (hunk 2 above)
Comment (cbarria) [reply fleet/lib/rook-ceph-conf/charts/pillan/templates/cephnfs-rsphome.yaml]: this was changed 👍🏻

**Pull request merged**